### DISCUSSION TITLE
Misleading notation (Task 3)

### DISCUSSION THREAD

**Discussion by JeppeKlitgaard** [Part-II-Computational-Physics/.github#49] [General]
It is entirely possible that I have managed to confuse myself, but I assume that the extremum $h << 1$ is to be taken as the limiting case for $h → 0$, which is not obvious from the notation used. $h << 1$ would indicate a strongly negative value for $h$. Physically I can't see a reason why $h$ should be constrained to positive values as it would make sense to allow anti-aligned external magnetic fields to be valid inputs as well.

I suggest changing $h << 1$ to $h ≈ 0$ or $|h| << 1$.

**Comment by JNHallen** [comment]
Yes what you want to check for is $0 \leq h \ll 1$.
